build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -0,0 +1,230 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.DotNet.Cli.Build.Framework;
+using Microsoft.DotNet.InternalAbstractions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using static Microsoft.DotNet.Cli.Build.Framework.BuildHelpers;
+using static Microsoft.DotNet.Cli.Build.FS;
+
+namespace Microsoft.DotNet.Cli.Build
```

Comment:
Would these namespaces change as well to `Microsoft.DotNet.Host.Build`? We now have multiple `CompileTargets` now for `build.cmd -Targets` -- btw, does this still work?